crypto/cipher_extra/e_aes_cbc_hmac_sha256.c

```diff
@@ -0,0 +1,375 @@
+/*
```

Comment:
I believe that comments from `crypto/cipher_extra/e_aes_cbc_hmac_sha1.c` also applies here.

Reply:
Not sure what this comment means.